oak_enclave_runtime_support/Cargo.toml

```diff
@@ -5,6 +5,10 @@ authors = ["Andri Saar <andrisaar@google.com>"]
 edition = "2021"
 license = "Apache-2.0"
 
+[features]
```

Comment:
I guess this should be removed now with the updated approach.

Reply:
We'll need to keep it around for a bit, until we can remove the global alloc allocator declaration in this library here. We can do so once other teams rely on it's behavior.

The SDK just imports the allocator struct, and disables the feature that sets the global alloc in this crates.